inc/orderinjection.class.php

```diff
@@ -32,15 +32,21 @@
 }
 
 class PluginOrderOrderInjection extends PluginOrderOrder implements PluginDatainjectionInjectionInterface {
-   public function __construct() {
+   function __construct() {
```

Comment:
I just tried to have functions declared the same way they are in all such inc/XXXXXinjection.class.php in datainjection or other plugins. Only in those 2 files in order plugin did I find the public visibily stated. But since I'm no coder, just keep it if you prefer.